src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ButtonAccessibleObjectTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+using Xunit;
+using static Interop.UiaCore;
+
+namespace System.Windows.Forms.Tests
+{
+    public class ButtonAccessibleObjectTests
+    {
+        [WinFormsFact]
+        public void CheckBoxAccessibleObject_Ctor_NullControl_ThrowsArgumentException()
+        {
+            Assert.Throws<ArgumentException>(() => new Button.ButtonAccessibleObject(null));
+        }
+
+        [WinFormsFact]
+        public void ButtonAccessibleObject_Constructor_InitializesOwner()
+        {
+            using var button = new Button();
+            var buttonAccessibleObject = new Button.ButtonAccessibleObject(button);
+
```

Comment:
```suggestion
            Assert.True(list.IsHandleCreated);

```

applies to all accessibility-concerned tests